theanomodels/utils/misc.py

```diff
@@ -54,10 +54,7 @@ def displayTime(event, start, end, reportingIn='seconds'):
     print '\t\t<',event,'> took ',time_taken,' ',reportingIn
 
 def getPYDIR():
-    for k in sys.path:
-        if 'theanomodels' in os.path.basename(k):
-            return k.split('theanomodels')[0]+'theanomodels'
-    assert False,'Should not reach here, directory <theanomodels> expected in PYTHONPATH.'
+    return os.getcwd()
```

Comment:
yes, its used in other models. i think we should leave it for the moment. ill check some of the other packages and take it out later if necessary.